### PR TITLE
[My Items] Make folder name customizable

### LINKED ISSUE #4626: [My Items] Customizable folder name on install
**Is your feature request related to a problem? Please describe.**
Eh. Certain instances of Open MCT would like to have the ability to have the my items folder name be customizable.

**Describe the solution you'd like**
When installing "My Items Plugin" pass in a name.

**Describe alternatives you've considered**
Currently you can change after an instance is brought up.

**Additional context**
NADA

### PR DESCRIPTION
**Pull request by jvigliotta**
### All Submissions:

Added ability to pass in a name to the MyItemsPlugin which would be used over the default "My Items" name on fresh installs of Open MCT.

closes #4626 

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Unit tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue?

### CODE REVIEW

File: src/plugins/myItems/myItemsInterceptor.js [hunk 1]
```diff
@@ -22,11 +22,11 @@
 
 import { MY_ITEMS_KEY } from "./createMyItemsIdentifier";
 
-function myItemsInterceptor(identifierObject, openmct) {
+function myItemsInterceptor(identifierObject, name, openmct) {
 
     const myItemsModel = {
         identifier: identifierObject,
-        "name": "My Items",
+        name,
```
Comment (nikhilmandlik) [inline src/plugins/myItems/myItemsInterceptor.js:29]: to be consistent can we convert all keys to be not a string?
eg: `"type":` -> `type` etc

**Review by nikhilmandlik — CHANGES_REQUESTED** [review]
minor change else LGTM

File: src/plugins/myItems/myItemsInterceptor.js (hunk 1 above)
Comment (jvigliotta) [reply src/plugins/myItems/myItemsInterceptor.js:29]: I don't know why you hate strings so much but ok... : )

**Review by nikhilmandlik — APPROVED** [review]

File: src/plugins/myItems/myItemsInterceptor.js [hunk 2]
```diff
@@ -22,7 +22,7 @@
 
 import { MY_ITEMS_KEY } from "./createMyItemsIdentifier";
 
-function myItemsInterceptor(identifierObject, name, openmct) {
+function myItemsInterceptor(identifierObject, openmct, name) {
```
Comment (nikhilmandlik) [inline src/plugins/myItems/myItemsInterceptor.js:25]: `myItemsInterceptor(identifierObject, openmct, name) {` -> `myItemsInterceptor(openmct, identifierObject, name) {` 😄

Comment (jvigliotta) [reply src/plugins/myItems/myItemsInterceptor.js]: hehe

**Review by nikhilmandlik — APPROVED** [review]
LGTM

**Pull request merged**